sidebars.ts

```diff
@@ -864,6 +864,11 @@ const sidebars: SidebarsConfig = {
           type: "doc",
           id: "server-core/java-core",
         },
+        {
+          type: "doc",
+          id: "server-core/java/migration",
+          label: "Java Core Migration Guide",
+        },
```

Comment:
Java should be a folder, with sdk docs as the first entry and migration guide next